examples/c/ksyscall.bpf.c

```diff
@@ -0,0 +1,27 @@
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+#define TASK_COMM_LEN 16
+
+SEC("ksyscall/kill")
+int BPF_KSYSCALL(entry_probe, pid_t tpid, int sig) 
+{
+
+	if (sig == 0) {
+		/**
+			If sig is 0, then no signal is sent, but existence and permission
+			checks are still performed; this can be used to check for the
+			existence of a process ID or process group ID that the caller is
+			permitted to signal.
+		*/
+		return 0;
+	}
+
+	bpf_get_current_comm(&comm, sizeof(comm));
```

Comment:
you lost `comm` variable somewhere along the way? see https://github.com/libbpf/libbpf-bootstrap/actions/runs/4643677178/jobs/8233653034?pr=167

Reply:
Apologies! Instead of removing the initialization, I've mistakenly removed the declaration also. Will get back with the changes